src/order-requests/order-requests.controller.ts

```
@@ -1,14 +1,146 @@
+  const user = req.user; // 요청한 사용자 정보 가져오기
+
+  if (!user) {
+    throw new UnauthorizedException('인증되지 않은 사용자입니다.');
+  }
+
+  const orderRequest = await this.orderRequestsService.getOrderRequestById(orderRequestId);
+  if (!orderRequest) {
+    throw new NotFoundException('주문 요청을 찾을 수 없습니다.');
+  }
+
+  if (orderRequest.requesterId !== user.userId) {
```

ESLint

[error] Insert `··`

(prettier/prettier)